app-accessibility/speech-dispatcher/files/speech-dispatcher-0.11.1-include-pthread_h.patch

```diff
@@ -0,0 +1,22 @@
+This fixes pthread_mutex_t being undefined on musl libc.
```

Comment:
This is upstream now: https://github.com/brailcom/speechd/commit/17e4038948f01e389cbd7fedbe3f0eb54ad3292d

What about the other?

Reply:
well kind of https://github.com/brailcom/speechd/commit/57e2c131f03ae8244b0478e449d86dea7d5724d8

Reply:
updated